### PR TITLE
Updated everything to Vulkano 0.27.1

### PR DESCRIPTION
**Pull request by ZakStar17**
### Hello!

When I first started learning about Vulkano, I found this guide actually helpful and I wanted to take some time to update it a bit. I  also plan to change it in a way that more people could use Vulkano as a efficient high level wrapper without much computer graphics knowledge.

### Nov 25, 2021 - Updated everything to Vulkano 0.26.0
- Updated Cargo.toml.
- Changed Rust edition to 2021.
- Updated the examples and the guide, until the windowing part (that doesn't have examples yet).

### Dec 12, 2021 - Updated everything to Vulkano 0.27.1
- Same as the last update, only to a newer version.
- Made the triangle viewport fixed in the pipeline.

### Next steps
- Update the guide be easier to incorporate Vulkano in a bigger project.
- Make the guide consistent with the Vulkano examples.
- Add sections for windowing, texturing, 3d.
- Make the guide more friendly for someone that is newer to computer graphics and doesn't understand everything that happens at a lower level.

### CODE REVIEW

**Comment by Eliah-Lakhin** [comment]
@ZakStar17 Thank you so much for your help in improving our website, and I'm very apologize for the late review.

Our website was in tremendously outdated state, and now thanks to your work it's **much** better.

Unfortunately I have limited time in reviewing our infrastructure, and I usually more focused on a primary repository. But I'm going to improve this situation and will take closer look at all PRs including vulkano-www.

Would you be interested in continuous improving of the website content and guides? PRs will be fast forwarded quicker, and maybe we can even make you a moderator of this repository if you are fine about it.

**Comment by ZakStar17** [comment]
@Eliah-Lakhin Many thanks to you for approving my pull request.

As you stated in your [Organization and Planning](https://github.com/vulkano-rs/vulkano/issues/1804) post, I also think its necessary to regularly improve the website and create other tutorials and guides from time to time. Vulkano has a really friendly high-level design, but the lack of guidance makes Vulkano hard to follow for begginers without previous Vulkan/graphics programing knowledge.

I would really like to continue improving the website and other content such as documentation, and I look forward to help improving the overall experience with the library on itself. I also don't mind helping review pull requests, although I may sometimes not have much time to do that.

On a side note, I mostly work on graphics programming as a hobby, so I don't have that much thecnical understanding of the subject. I would really appreciate if someone could review the contents of the website to see if they are entirely correct, as I partially changed some parts, and small mistakes could ruind a bit the experience.

**Comment by Eliah-Lakhin** [comment]
@ZakStar17 I'm glad to have you and @hakolao as regular Website maintainers. I just added to both of you the Write access to this repository. Please approve my GitHub invitation, it should arrive in your email inbox.

From now on you can review and merge Pull Requests, and also push commits to the master branch directly too. Once the commit arrives in the master branch, the vulkano.rs website reflects these changes almost instantly. So, please be careful in changing things.

I don't want to limit you, guys, in a way you want to evolve the website, but I suggest sticking with the rule in adding new changes through the Pull Requests exclusively, and for your own PRs to merge them only when you get approval from someone else of the Vulkano Community. e.g. I just approved Zakhar's #99, so he can proceed with the PR merging. Of course, there are exceptions to this rule, when you need to fix something minor.

Just some ideas for you in which way we can improve the website:

 1. Okko suggested creating a Discord server as a Gitter Chat replacement. I like this idea. And we can add a permanent invite link on our Website too.
 2. Of course, keep improving the Tutorial. Maybe eventually turn it into a book that would cover a wider range of Vulkano features and technical topics.
 3. We can create an additional page with projects using Vulkano(aka "Showcase"). On this page, we will have more space to add a detailed description of the projects.

Good luck!
